liberty-maven-plugin/src/it/tests/pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>assembly-artifact-no-version-it</module>
         <module>ear-project-it</module>
         <module>install-features-it</module>
+        <module>install-features-dependencies-it</module>
```

Comment:
This should be added to the Open Liberty profile since it should only run on Open Liberty.